*** tests/A-13-17-M-ok.out	2022-11-03 21:19:15.000000000 +0000
--- tests/A-13-17-M-ok.outhyp	2022-11-13 23:49:37.000000000 +0000
***************
*** 31,37 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 31,90 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969014' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: O terminal já estava ligado.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969015' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 39,48 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 92,102 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 55,61 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 109,116 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969014' não existe.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 63,72 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 118,128 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Opção inválida!
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 79,85 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 135,142 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava ligado.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 87,96 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 144,154 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969015' não existe.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 103,109 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 161,168 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 111,120 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 170,180 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 144,150 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 204,211 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969014' não existe.
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 152,161 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 213,236 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 185,191 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
--- 260,280 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): Iniciar comunicação: Operação inválida: O terminal '969014' não existe.
! OPERAÇÕES DO TERMINAL
! 1 - Ligar
! 2 - Colocar no silêncio
! 3 - Desligar
! 4 - Adicionar amigo
! 5 - Retirar amigo
! 6 - Pagamento
! 7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
! 11 - Mostrar comunicação em curso
! 0 - Sair
! Escolha uma opção: Opção inválida!
! OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
  3 - Desligar
***************
*** 193,202 ****
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 282,292 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
! 8 - Enviar comunicação de texto
! 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: O terminal já estava ligado.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 230,238 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|PLATINUM|YES|4|520|100
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
! CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|20
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
--- 320,328 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
! CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
