*** tests/A-22-03-M-ok.out	2022-10-31 09:22:07.000000000 +0000
--- tests/A-22-03-M-ok.outhyp	2022-11-13 23:49:43.000000000 +0000
***************
*** 79,85 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
--- 79,86 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: Mostrar comunicações recebidas por um cliente: Operação inválida: O terminal 'cli201' não existe.
! CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
