*** tests/A-19-07-M-ok.out	2022-10-30 22:21:45.000000000 +0000
--- tests/A-19-07-M-ok.outhyp	2022-11-02 19:42:15.000000000 +0000
***************
*** 33,38 ****
--- 33,40 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: O terminal já estava desligado.
***************
*** 44,49 ****
--- 46,53 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
***************
*** 59,64 ****
--- 63,70 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: O terminal já estava desligado.
***************
*** 70,75 ****
--- 76,83 ----
  5 - Retirar amigo
  6 - Pagamento
  7 - Consultar saldo
+ 8 - Enviar comunicação de texto
+ 9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: GESTÃO DE TERMINAIS
